Tools/autotest/common.py

```diff
@@ -778,12 +803,16 @@ def arm_vehicle(self, timeout=20):
                 return True
         raise AutoTestTimeoutException("Failed to ARM with mavlink")
 
-    def disarm_vehicle(self, timeout=20):
+    def disarm_vehicle(self, timeout=20, force=False):
         """Disarm vehicle with mavlink disarm message."""
         self.progress("Disarm motors with MAVLink cmd")
+        p2 = 0
+        if force:
+            magic_force_disarm_value = 21196
+            p2 = magic_force_disarm_value
```

Comment:
Just noting that this is the deprecated way of force disarm.

Reply:
Remind me which mechanism replaced it?

Reply:
MAV_CMD_DO_FLIGHTTERMINATION